plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -113,6 +117,36 @@ export default function transformProps(chartProps: ChartProps): EchartsTimeserie
       });
   });
 
+  annotationLayers.forEach((layer: AnnotationLayer) => {
+    const {
+      name,
+      color,
+      opacity: annotationOpacity,
+      width: annotationWidth,
+      show: annotationShow,
+      style,
+    } = layer;
+    if (isFormulaAnnotationLayer(layer) && annotationShow) {
```

Comment:
NIT: would flip the two as `annotationShow` is certainly cheaper to evaluate and js won't even call `isFormulaAnnotationLayer` if it's `false`